lib/view_model/wallet_creation_vm.dart

```diff
@@ -89,7 +89,7 @@ abstract class WalletCreationVMBase with Store {
     }
   }
 
-  DerivationInfo? getDefaultDerivation() {
+  DerivationInfo? getDefaultCreationDerivation() {
```

Comment:
is the derivation path for Electrum here correct?
Weren't your changes intended to remove the second `/0`